test/studies/shootout/submitted/knucleotide2.good

```diff
@@ -1,3 +1,12 @@
+knucleotide2.chpl:102: warning: ascii is deprecated - please use string.byte instead
```

Comment:
Seems like this will also mess up line based verification in the `knucleotide2.perfkeys` file?

Reply:
Oh wait, nevermind -- I see there is still a perfcompopts throwing `--no-warnings`

Reply:
I'm glad you answered that question, because I had the same one, but forgot to answer it for myself before opening the PR (though I had run testing and it passed).